lib/packwerk/node.rb

```diff
@@ -192,7 +195,7 @@ def parent_module_name(ancestors:)
       end
 
       def type(node)
-        node.type
+        node&.type
```

Comment:
Do you still need this now that you are handling `self::` explicitly? The only test that fails when I revert this change is the test that explicitly tests this change.

Reply:
ok!